cmd/main.go

```diff
@@ -38,6 +38,16 @@ func main() {
 		panic(fmt.Errorf("issues with environment sourcing: %s", err))
 	}
 
+	flags := config.NewFlags()
+
+	switch {
+	case flags.ShowVersion:
+		fmt.Printf("App Version: %s\n", config.Version)
```

Comment:
do we want the pure `fmt.Print` or is it better with the logger?

Reply:
I didn't use the logger because these are not logs, I didn't want any special formatting and I didn't want it to get written to a file on accident.